<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGestionPointDeVente</class>
 <widget class="QDialog" name="DialogGestionPointDeVente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des points de ventes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Libelle : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditLibellePdv"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="8">
    <widget class="QTableWidget" name="tableWidgetPointDeVente">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>145</number>
     </attribute>
     <column>
      <property name="text">
       <string>Libellé</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Activité</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom du dirigeant</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Prénom du dirigeant</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Téléphone</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rue</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Code Postal</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ville</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Activité :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditActivitePdv"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Nom du dirigeant :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNomDirigeantPdv"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Prénom du dirigeant : :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPrenomDirigeatPdv"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Téléphone :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTelephonePdv"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Rue :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditRuePdv"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Code postal :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCodePostalPdv"/>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Ville :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditVillePdv"/>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="pushButtonAjoutPdv">
       <property name="text">
        <string>Ajouter un point de vente</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSupprPdv">
       <property name="text">
        <string>Supprimer un point de vente</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogGestionPointDeVente</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogGestionPointDeVente</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
